### PR TITLE
Add SQS support (AWS Collector) to kube-metrics-adapter

### PR DESCRIPTION
**Pull request by adeelx**
See more: https://github.com/zalando-incubator/kube-metrics-adapter#aws-collector

### CODE REVIEW

File: kube-metrics-adapter/templates/deployment.yaml [hunk 1]
```diff
@@ -49,14 +53,24 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: tmp
+        - name: ssl-certs
```
Comment (sancyx) [inline kube-metrics-adapter/templates/deployment.yaml]: Could you please make this option as well `if .Values.aws.enable`

File: kube-metrics-adapter/templates/deployment.yaml [hunk 2]
```diff
@@ -49,14 +53,24 @@ spec:
...
 {{ toYaml .Values.affinity | indent 8 }}
+    {{- end }}
+    {{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
       volumes:
       - name: tmp
         emptyDir: {}
+      - name: ssl-certs
```
Comment (sancyx) [inline kube-metrics-adapter/templates/deployment.yaml]: Could you please make this option as well `if .Values.aws.enable`

Comment (adeelx) [reply kube-metrics-adapter/templates/deployment.yaml]: Done, Thank you for the review.

File: kube-metrics-adapter/templates/deployment.yaml (hunk 1 above)
Comment (adeelx) [reply kube-metrics-adapter/templates/deployment.yaml]: Done, Thank you for the review.

**Comment by adeelx** [comment]
@sancyx / @ahma / @kozmagabor Could you kindly review once more? The requested changes have been integrated.

**Comment by adeelx** [comment]
Thank you @sancyx / @ahma / @kozmagabor, could one of you kindly merge?

**Pull request merged**